### ISSUE TITLE
[Question]How to align a component to other components in Grid?

### ISSUE THREAD

**Issue by tzengshinfu** [stsrki/Blazorise#1365]
**Describe the bug**
I tried to move the Addon conponent to right to align other components with the WithOffset attribute, but it still keeps remaining space.
![actual](https://user-images.githubusercontent.com/2170033/95285209-d82ce380-0892-11eb-9da9-75bcef6465e5.png)

**To Reproduce**
![Image 007](https://user-images.githubusercontent.com/2170033/95285175-b92e5180-0892-11eb-8bcb-eafa486e59f7.png)

**Expected behavior**
![except](https://user-images.githubusercontent.com/2170033/95285226-e1b64b80-0892-11eb-9d12-941da8dab985.png)

**Comment by stsrki** [comment]
I don't think this is possible with any framework without manually adjusting the CSS for your own scenario. That being said I managed to do it close enough with different approach

```cshtml
<Row>
    <Column ColumnSize="ColumnSize.IsHalf">
    </Column>
    <Column ColumnSize="ColumnSize.IsHalf">
        <Row>
            <Column ColumnSize="ColumnSize.Is5" Padding="Padding.Is0.FromRight">
                <TextEdit />
            </Column>
            <Column ColumnSize="ColumnSize.Is2" Padding="Padding.Is0.OnX">
                <Button Color="Color.Primary" Block="true">...</Button>
            </Column>
            <Column ColumnSize="ColumnSize.Is5" Padding="Padding.Is0.FromLeft">
                <TextEdit />
            </Column>
        </Row>
    </Column>
</Row>
```

I hope it helps.

**Comment by tzengshinfu** [comment]
You're right.
I have added the margin-left attribute to move it to the right to align to other components.
![Image 003](https://user-images.githubusercontent.com/2170033/95302260-48982c80-08b4-11eb-935e-5ce6b22cde0c.png)

When screen size changed, it wraps into new line...
![Image 004](https://user-images.githubusercontent.com/2170033/95302977-37035480-08b5-11eb-8214-343e643c4977.png)


I tried your resolution, it works perfectly. I'll follow your advice.

Thanks your helping.
